main/ws2812_control.c

```diff
@@ -34,7 +34,10 @@ void ws2812_control_init(gpio_num_t led_gpio)
   ESP_ERROR_CHECK(rmt_driver_install(config.channel, 0, 0));
 }
 
-void ws2812_write_leds(struct led_state new_state) {
+void ws2812_write_leds(uint32_t led_state) {
```

Comment:
this is probably the only breaking change. the struct with the colors was mentioned in a few files and i couldn't get the compiler to play nicely so i yolo'd it

Reply:
won't kill me, it's not like the struct was doing much to start with given that there's only one led